OSEHRA/VistA
# Ensure that the package mapping is consistent

Previously, the ICR parser would modify the global PACKAGE_MAP in
UtilityFunctions but only if html files were generated. Depending on how
scripts were launched, the WebPageGenerator may or may not use this
updated information.

To test:
1. Run *just* the PDF ICR tests
2. Run WebPageGenerator. Ensure that from the Global page for IBA(355.32,
the link to the Subscribing Package(s) in the ICR table work
